inc/ThirdParty/Themes/Divi.php

```diff
@@ -108,6 +113,29 @@ public function add_divi_to_description( $disable_youtube_lazyload ) {
 		return $disable_youtube_lazyload;
 	}
 
+	/**
+	 * Disables setting explicit dimensions on images where Divi calculates height as percentage.
+	 *
+	 * @since 3.8.2
+	 *
+	 * @param array $images The array of images selected for adding image dimensions.
+	 *
+	 * @return array The array without images using data-height-percentage.
+	 */
+	public function disable_image_dimensions_height_percentage( array $images ) {
+		if ( ! self::is_divi() ) {
+			return $images;
+		}
+
+		foreach ( $images as $key => $image ) {
```

Comment:
I think this check is case insensitive, can we check the following proposed solution if it's faster or not
```
return array_filter($images, function($img) {
	return ! preg_match('#data-height-percentage#i', $img);
});
```

Reply:
Divi specifically defines this, so while it's pretty certain that it will always come to us lowercase, I agree that making case insensitive is safer.

Current solution adding `strtolower()` runs on average half the time as the regex in my tests.

Reply:
Great